File: backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/LwjglAudio.java

package com.badlogic.gdx.backends.lwjgl.audio;

import com.badlogic.gdx.Audio;
import com.badlogic.gdx.backends.lwjgl.LwjglApplication;

public interface LwjglAudio extends Audio {

    /**
     * Updates audio state (usually called every frame by the {@link LwjglApplication})
     */
    void update();
}

File: backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/LwjglAudio.kt
package com.badlogic.gdx.backends.lwjgl.audio

import com.badlogic.gdx.Audio
import com.badlogic.gdx.backends.lwjgl.LwjglApplication

interface LwjglAudio : Audio {
    /**
     * Updates audio state (usually called every frame by the [LwjglApplication])
     */
    fun update()
}